packages/backend/src/modules/task/task.model.ts

```diff
@@ -1,30 +1,28 @@
 import mongoose from 'mongoose'
 
-const { ObjectId } = mongoose.Schema
+const { ObjectId } = mongoose.Types
 
 const taskSchema = new mongoose.Schema({
-  _id: ObjectId,
   title: String,
   description: String,
   imageCoverId: String,
   attachments: [
     {
-      _id: ObjectId
+      attachments_id: ObjectId
```

Comment:
leave it as it is? Why this change?

Reply:
previously it did not correctly generate mongoose IDs